MediaTracker/app/src/main/java/com/izor066/android/mediatracker/api/model/Book.java

```diff
@@ -0,0 +1,55 @@
+package com.izor066.android.mediatracker.api.model;
+
+/**
+ * Created by igor on 7/11/15.
+ */
+public class Book {
+
+    private final String title;
+    private final String author;
+    private long datePublished;
```

Comment:
All of the above fields can be `final`. We use final in order to promote immutability.